### PR TITLE
Midterm blog

### PR DESCRIPTION
**Pull request by YQXie99**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [ ] Add Blog Header, including title, author, date.
e.g.

``` md
---
title: "'Score Your Face Photo' a ML&Wechaty practice"
author: huyingxi
image: /assets/2020/your-blog-title/teaser.jpg
---

YOUR_FIRST_SENTENCE_SHOULD_BE_A_SUMMARY_FOR_YOUR_POST

```

- [ ] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [ ] Embed the photo & video before publishing, save all external file to the blog `/assets/2020` directory.
- [ ] Add teaser image for the blog
- [ ] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.js.org/blob/master/_developers/YOUR_GITHUB_USERNAME.yml> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

[See more](https://github.com/wechaty/wechaty.js.org#contribute-guideline)

### CODE REVIEW

File: _posts/2020-08-19-puppet-work-midterm.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+title: "暑期2020 [基于开放 API 封装 Wechaty 接口下的企业微信聊天机器人] POC 成果展示"
+date: 2020-08-19 22:00 +0800
+author: Sapio-S
+categories: project
+image: /assets/2020/work-puppet/pic.jpg
```
Comment (lijiarui) [inline _posts/2020-08-19-puppet-work-midterm.md]: Where is this image?

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change as I request.

File: _posts/2020-08-19-puppet-work-midterm.md (hunk 1 above)
Comment (YQXie99) [reply _posts/2020-08-19-puppet-work-midterm.md]: 图片应该在前一个pull request里面（两个blog共用一张图）

Comment (lijiarui) [reply _posts/2020-08-19-puppet-work-midterm.md]: Each blog should have it's separate image. Please upload a new image for this blog.

Comment (YQXie99) [reply _posts/2020-08-19-puppet-work-midterm.md]: 好的，已经重新添加了图片。麻烦您了！

File: _posts/2020-08-19-puppet-work-midterm.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
+---
+title: "暑期2020 [基于开放 API 封装 Wechaty 接口下的企业微信聊天机器人] POC 成果展示"
+date: 2020-08-19 22:00 +0800
+author: Sapio-S
+categories: project
+image: /assets/2020/work-puppet/pic2.jpg
+tags:
+  - featured
+  - Soc2020
```
Comment (lijiarui) [inline _posts/2020-08-19-puppet-work-midterm.md]: Please add soc tag as well and change `Soc2020` to `soc2020`

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change as I request and resolve the conflict, thanks!

**Comment by lijiarui** [comment]
Please make CI green and resolve the conflict, and then I can review this blog.

**Review by YQXie99 — COMMENTED** [review]
抱歉我发现产生冲突的文件无法删除……请问有没有别的办法可以删除，还是只能重新提交一个pull request了呢？

**Comment by lijiarui** [comment]
Create a new one is a good option.

**Comment by lijiarui** [comment]
@wj-Mcat  Please help @Sapio-S  posting his blog.

**Comment by wj-Mcat** [comment]
Please close this issue and create new one.  If you have not completed this issue in next week, we will close this issue. Thanks for your attention.

**Pull request closed without merging**